cmd/query/app/handler.go

```diff
@@ -113,6 +114,9 @@ func NewAPIHandler(spanReader spanstore.Reader, dependencyReader dependencystore
 	if aH.logger == nil {
 		aH.logger = zap.NewNop()
 	}
+	if aH.tracer == nil {
+		aH.tracer = opentracing.GlobalTracer()
```

Comment:
would you be ok if I use
`opentracing.NoopTracer{}`
instead here rather than GlobalTracer?